File: 4AL17IS014_DIKSHIT_KOTIAN/Sharan_coding-challenge_1/Armstrong.py
n = int(input("Input an integer"))
sum1 = 0
count=0
temp = n
t=n
while t>0:
    t=t//10
    count=count+1
while temp > 0:
   rem = temp % 10
   sum1 += rem ** count
   temp =temp// 10
print("Is an Armstrong number?")
if n == sum1:
   print("Yes")
else:
   print("No")

File: 4AL17IS024_MAYURESH_KUNDER/SharanSir_Coding_Challenge/Coding_Challenge_1/Armstrong.java
import java.util.*;
import java.lang.*;
public class MK {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        int input = sc.nextInt();
        int len = String.valueOf(input).length();
        int temp = input,rem = 0,sum = 0;
        while(temp>0)
        {
             rem = temp%10;
             sum += Math.pow(rem, len);
             temp /= 10;
        }
        if(input == sum)
        System.out.println("Yes");
        else
            System.out.println("No");
    }
}
